src/libraries/Common/tests/System/IO/ReparsePointUtilities.cs

```diff
@@ -44,10 +96,20 @@ public static bool CreateSymbolicLink(string linkPath, string targetPath, bool i
         }
         symLinkProcess.StartInfo.UseShellExecute = false;
         symLinkProcess.StartInfo.RedirectStandardOutput = true;
+
+        // symLinkProcess.StartInfo.RedirectStandardError = true;
+        // string output = "";
+        // symLinkProcess.OutputDataReceived += new DataReceivedEventHandler((sender, e) => { output += e.Data; });
+
         symLinkProcess.Start();
 
+        // symLinkProcess.BeginOutputReadLine();
+        // string error = symLinkProcess.StandardError.ReadToEnd();
+        // Debug.WriteLine($"{error} \n {output}");
+
```

Comment:
I guess it was not intended to include it

```suggestion
```

Reply:
If mklink fails in future for some reason this is what you'd uncomment, to better see it, but yup I'll remove it.